server/svix-server/config.default.toml

```diff
@@ -39,6 +39,12 @@ log_format = "default"
 # always sending. If the OpenTelemetry address is not set, this will do nothing.
 # opentelemetry_sample_ratio = 1.0
 
+# The Sentry DSN to use for error reporting. Disabled when omitted/null
+# sentry_dsn = "https://somedsn.ingest.sentry.io/12345"
+
+# The environment name to report to Sentry (when Sentry is enabled). Uses the Sentry default when omitted/null
+# sentry_environment = "development"
```

Comment:
I think we already have an `env` parameter that you can use? Though even if not let's make it a general config.

Reply:
We have it in Python for sure (and it should be an enum btw, unlike python)

Reply:
I renamed this to `environment`, and made it a `dev`/`staging`/`prod` enum